amundsen_application/static/js/components/TableDetail/RequestMetadataForm/index.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import './styles.scss';
+
+import { sendNotification } from 'ducks/notification/api/v0';
+import { NotificationType } from 'interfaces/Notifications';
+import { AvatarLabelProps } from 'components/common/AvatarLabel';
+import { GlobalState } from 'ducks/rootReducer';
+import { connect } from 'react-redux';
+
+interface StateFromProps {
+  userEmail: string;
+  tableName: string;
+  tableOwners: Array<String>;
+}
+
+type RequestMetadataProps = StateFromProps;
+
+interface RequestMetadataState {
+  isOpen: boolean,
+}
+
+export class RequestMetadataForm extends React.Component<RequestMetadataProps, RequestMetadataState> {
+  public static defaultProps: RequestMetadataProps = {
+    userEmail: '',
+    tableName: '',
+    tableOwners: [],
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpen: true,
+    };
+  }
+
+  toggle = () => {
+    this.setState({ isOpen: !this.state.isOpen });
+  }
+
+  submitForm = (event) => {
+    event.preventDefault();
+    const form = document.getElementById("RequestForm") as HTMLFormElement;
+    const formData = new FormData(form);
+    const recipients = String(formData.get('recipients')).split(",");
+    const sender = String(formData.get('sender'));
+    const descriptionRequested = formData.get('table-description') === "on" ? true : false;
+    const fieldsRequested = formData.get('column-description') === "on" ? true : false;
+    const comment = String(formData.get('details'));
+    sendNotification(
+      recipients,
+      sender,
+      NotificationType.REQUESTED,
+      {
+        comment,
+        resource_name: this.props.tableName,
+        resource_url: window.location.href,
+        description_requested: descriptionRequested,
+        fields_requested: fieldsRequested,
+      }
+    )
+  };
+
+  render() {
+    const expandedClass = this.state.isOpen ? 'expanded' : 'collapsed';
+    return (
+      <div className={`request-component ${expandedClass}`}>
+        <div className="form-section request-header">
+          <h3 className="title">Amundsen Resource Request</h3>
```

Comment:
We'll want to separate all hardcoded english text out into a `constants.ts` file, the import the constants here and use them. 

This will help us in the future when we implement some system for folks to configure custom text, either for their own UI needs or to support translating the application's text into another language.